docs/reference/requirement-sets/shared-runtime-requirement-sets.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Shared runtime requirement sets
+description: 'Specifies the platforms and Office hosts that support the SharedRuntime APIs.'
+ms.date: 01/21/2020
+ms.prod: non-product-specific
+localization_priority: Normal
+---
+
+# Shared runtime requirement sets
+
+Requirement sets are named groups of API members. Office Add-ins use requirement sets specified in the manifest or use a runtime check to determine whether an Office host supports APIs that an add-in needs. For more information, see [Office versions and requirement sets](/office/dev/add-ins/develop/office-versions-and-requirement-sets).
+
+Parts of an Office Add-in that run JavaScript code, such as task panes, function files launched from add-in commands, and Excel custom functions, can share a single JavaScript runtime. This enables all the parts to share a set of global variables, to share a set of loaded libraries, and to communicate with each other without having to pass messages through a persisted storage.
+
+The following table lists the SharedRuntime 1.1 requirement set, the Office host applications that support that requirement set, and the build or version numbers for the Office application.
+
+|  Requirement set  |  Office 2013 (or later) on Windows<br>(one-time purchase) | Office on Windows<br>(connected to Office 365 subscription)   |  Office on iPad<br>(connected to Office 365 subscription)  |  Office on Mac<br>(connected to Office 365 subscription)  | Office on the web  | Office Online Server |
+|:-----|:-----|:-----|:-----|:-----|:-----|:-----|
+| SharedRuntime 1.1  | N/A | Version 2001 (Build 12527.20000) or later | N/A | 16.34 or later | January 2020 | N/A |
+
+To find out more about versions, build numbers, and Office Online Server, see:
+
+- [Version and build numbers of update channel releases for Office 365 clients](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
+- [What version of Office am I using?](https://support.office.com/article/What-version-of-Office-am-I-using-932788b8-a3ce-44bf-bb09-e334518b8b19)
+- [Where you can find the version and build number for an Office 365 client application](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
+- [Office Online Server overview](/officeonlineserver/office-online-server-overview)
+
+## Office Common API requirement sets
+
+For information about Common API requirement sets, see [Office Common API requirement sets](office-add-in-requirement-sets.md).
```

Comment:
Should you also update the **Office Common API requirement sets** article to add a subsection for this new **SharedAPI** requirement set? -- for example, like this section for the **IdentityAPI** (another *common* API):

![image](https://user-images.githubusercontent.com/3019529/73017095-0a191b00-3dd4-11ea-907f-808ffa58e809.png)

Reply:
Good catch. Just added that.